eng/pipelines/templates/jobs/build.yml

```diff
@@ -76,6 +76,17 @@ jobs:
     inputs:
       summaryFileLocation: $(CoverageFile)
 
+  - task: Powershell@2
+    displayName: "(Native AOT) Build module"
+    condition: and(succeeded(), ne(variables['NoPackagesChanged'],'true'), eq('${{ parameters.OSName }}', 'linux'), eq(variables['Architecture'], 'x64'))
```

Comment:
Are you scoping it to just linux-x64 because it is time consuming? Eventually we will want it for all architectures so why not validate all now?

Reply:
Agreed, let’s run it for all architectures. Since jobs run in parallel across arch hosts, the overall CI time should be the same whether we use one host or all hosts.